cmd/sifnoded/genaccounts.go

```diff
@@ -228,6 +229,109 @@ contain valid denominations. Accounts may optionally be supplied with vesting pa
 	return cmd
 }
 
+func AddFaucetBalance(
+	ctx *server.Context, cdc *codec.Codec, defaultNodeHome, defaultClientHome string,
+) *cobra.Command {
+
+	cmd := &cobra.Command{
+		Use:   "add-faucet [coin][,[coin]]",
+		Short: "Add a faucet to the chain",
+		Long: `Add a genesis account to genesis.json. The provided account must specify
+the account address or key name and a list of initial coins. If a key name is given,
+the address will be looked up in the local Keybase. The list of initial tokens must
+contain valid denominations. Accounts may optionally be supplied with vesting parameters.
+`,
+		Args: cobra.ExactArgs(1),
+		RunE: func(cmd *cobra.Command, args []string) error {
+			config := ctx.Config
+			config.SetRoot(viper.GetString(cli.HomeFlag))
+
+			//addr,_:= sdk.AccAddressFromBech32("sif17s95c5jpc6x2l3edwh4dm8yhac68yru7a7kr3x")
+			addr := faucet.GetFaucetModuleAddress()
+			coins, err := sdk.ParseCoins(args[0])
+			if err != nil {
+				return fmt.Errorf("failed to parse coins: %w", err)
+			}
+
+			vestingStart := viper.GetInt64(flagVestingStart)
```

Comment:
This can be cleaned up a bit , I dont think the faucet needs to have vesting ,

Reply:
I agree. Removed